docs/content/configuration/csi_ug_config_create_replication.md

```diff
@@ -0,0 +1,46 @@
+# Creating a VolumeReplication
+
+Create a VolumeReplication yaml file to replicate a specific PersistentVolumeClaim (PVC).
+
+VolumeReplicationClass needs to be present before a VolumeSnapshot can be created. For more information, see [Creating a VolumeReplicationClass](csi_ug_config_create_vol_replicationclass.md).
+
+**Note:** Replication is also known as mirroring in many storage systems.
+
+When replicating a volume, be sure to follow all of the replication configurations, found in [Compatibility and requirements](../installation/csi_ug_requirements.md) before snapshot creation.
+
+1.  Replicate a specific  PersistentVolumeClaim (PVC) using the demo-replication.yaml.
+
+    For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
+
+    ```
+    apiVersion: replication.storage.openshift.io/v1alpha1
+    kind: VolumeReplication
+    metadata:
+      name: volumereplication-sample
+      namespace: default
+    spec:
+      volumeReplicationClass: volumereplicationclass-sample
+      replicationState: primary
+      dataSource:
```

Comment:
note that we are still [working on](https://github.com/csi-addons/volume-replication-operator/pull/109) adding an additional field, which we call [`targetVolumeHandle`](https://github.com/csi-addons/volume-replication-operator/blob/6462612ffb0d7c5852757fea15ccb99e996a310e/README.md#volumereplication)

Reply:
ack.
when relevant, please open a ticket with the specific changes that will be necessary.

Reply:
np